system-int-test/test/cmr/system_int_test/ingest/translation_test.clj

```diff
@@ -47,19 +47,28 @@
   <Visible>true</Visible>
   </Collection>")
 
+(defn- convert-to-sets
+  "Convert lists in the umm record to sets so order doesn't matter during comparison"
+  [record]
+  (-> record
```

Comment:
I understand your point. As time goes on, we might have to convert more things to sets like the one in generate-and-parse test.  It's just a place holder so that more things could be converted to sets.  Make sense?